jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/internal/JRubyExecDelegate.groovy

```diff
@@ -95,13 +96,19 @@ class JRubyExecDelegate implements JRubyExecTraits   {
                 "${k}" v
             }
             main 'org.jruby.Main'
+            // just keep this even if it does not exists
+            args '-I' + new File(gemDir, "gems/jar-dependencies-${JRubyExec.JAR_DEPENDENCIES_VERSION}/lib").absolutePath
```

Comment:
This line looks the same as [this line](https://github.com/jruby-gradle/jruby-gradle-plugin/pull/134/files#diff-017bc85ec183b742e079cba05ac40f66R215).

Will you please refactor this behavior into a method which both locations can call?